packages/v4/src/content/design-guidelines/components/drag-and-drop/drag.md

```diff
@@ -0,0 +1,72 @@
+---
+id: Drag and drop
+section: components
+---
+The **drag and drop** interaction can be used to reposition elements on screen into a layout that benefits the user. This gives the user more flexibility to arrange and/or group items without having to make code changes.
+
+## Usage
+### Simple list
+1. **Draggable area:** The drag and drop interaction can be triggered with click and hold over any part of the item. The `fa-grip` icon is used to show that the drag and drop interaction is available.
+
+<img src="./img/Simple-list-drag-1.png" width="588" alt="Showing the area in which an item can be dragged."/>
+
+2. **Bounding box:** Upon click & hold a `pf-color-blue-400` border will show the draggable area that is available. 
```

Comment:
https://github.com/patternfly/patternfly/blob/541f89d94d4921d5512e02b30f3e09ac01437281/src/patternfly/components/DragDrop/drag-drop.scss#L6

I would reference the CSS var here, instead of the sass var name, which is just equal to a hex value. That implies the border should always be whatever `--pf-global--active-color--100` is, including if we change our global active color from `blue-400` to `blue-300`, for example. It also works across light and dark theme (and other themes, presumably)

wdyt @mcarrano @mceledonia @mmenestr? Do we have a standard for what kind of value/var to reference in places like this?

```suggestion
2. **Bounding box:** Upon click & hold a `--pf-global--active-color--100` border will show the draggable area that is available. 
```

Reply:
Also just checked what we have in the sketch library, and we use the css var there, so I think it definitely makes sense to use it here too! Good catch